src/ui/server/controllers/insurance/account/sign-in/index.ts

```diff
@@ -46,14 +46,24 @@ export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.SIGN_IN.ROOT;
  * @param {Express.Response} Express response
  * @returns {Express.Response.render} Sign in page
  */
-export const get = (req: Request, res: Response) =>
-  res.render(TEMPLATE, {
+export const get = (req: Request, res: Response) => {
+  const flash = req.flash('successBanner');
+
+  let renderSuccessBanner = false;
+
+  if (flash.includes('newAccountVerified')) {
```

Comment:
very small comment:
this could be simplified into something like
```
let renderSuccessBanner = flash.includes('newAccountVerified') || false;
```

Reply:
Thanks @IgnatG - surprised I didn't think of that 🤦 will update this.